lms/djangoapps/instructor/tests/test_spoc_gradebook.py

```diff
@@ -90,6 +86,10 @@ class TestDefaultGradingPolicy(TestGradebook):
     Tests that the grading policy is properly applied for all users in the course
     Uses the default policy (50% passing rate)
     """
+    def test_response_code(self):
```

Comment:
Makes sense, I was on the fence about where to put it. Putting it in the setUp means both test cases will fail on setUp though, which I suppose isn't the worst thing.